src/Database/DBA.php

```diff
@@ -8,6 +8,7 @@
 
 use Friendica\Core\System;
 use Friendica\Util\DateTimeFormat;
+use mysqli;
```

Comment:
Now you can remove this line to pass the tests.

Reply:
Just a question, don't we still need it for the `new mysqli` part?

Reply:
It's already been added on line 15.

Reply:
Hmm, on other places. I revert then. And later we can choose how to reorganize (mostly alphabetically sorted) "imports".

Reply:
My IDE does it by alphabetical order.